Manito/Manito/Screens/Detail-Ing/DetailngCodebaseViewController.swift

```diff
@@ -323,34 +393,214 @@ final class DetailingCodebaseViewController: BaseViewController {
         }
     }
     
-    override func configUI() {
-        super.configUI()
-        setupText()
-    }
-    
     override func setupGuideArea() {
         super.setupGuideArea()
         guideButton.setImage(ImageLiterals.icMissionInfo, for: .normal)
         setupGuideText(title: TextLiteral.detailIngViewControllerGuideTitle, text: TextLiteral.detailIngViewControllerText)
     }
-
-    private func setupText() {
-        titleLabel.text = "애니또 팀"
-        periodLabel.text = "22.11.11 ~ 22.11.15"
-        statusLabel.text = "진행중"
-        missionContentsLabel.attributedText = NSAttributedString(string: "1000원 이하의 선물 주고 인증샷 받기")
-        manitteeLabel.text = "디너의 마니띠"
-        manitteeAnimationLabel.text = "호야"
+    
+    override func setupNavigationBar() {
+        super.setupNavigationBar()
+        let rightItem = makeBarButtonItem(with: exitButton)
+        navigationItem.rightBarButtonItem = rightItem
+    }
+    
+    private func setupLargeTitleToOriginal() {
```

Comment:
p3;

func 주석을 활용해서 private func 코드들을 정리하는건 어떨까요 ??



```suggestion
    }
    
    // MARK: - func
    
    private func setupLargeTitleToOriginal() {
```

Reply:
> p3;
> 
> func 주석을 활용해서 private func 코드들을 정리하는건 어떨까요 ??

4f6309c 좋습니다!

Reply:
> p3;
> 
> func 주석을 활용해서 private func 코드들을 정리하는건 어떨까요 ??

62d61b5 좋습니다!